### PR TITLE
Add AV1 Support

### PR DESCRIPTION
**Pull request by Sean-Der**
I tried to keep this as simple/terse as possible.

It supports two inbound Packetization formats. Media that is supplied temporal unit at a time, or OBU at a time.

### CODE REVIEW

File: src/av1rtppacketizer.cpp
Comment (paullouisageneau) [inline src/av1rtppacketizer.cpp:46]: You should check the size if `message` before accessing the data.

File: src/av1rtppacketizer.cpp
Comment (paullouisageneau) [inline src/av1rtppacketizer.cpp:125]: Mostly a nitpick, but using the vector range constructor for `binary` here would be more idiomatic than allocation then memcpy.

File: src/av1rtppacketizer.cpp
Comment (paullouisageneau) [inline src/av1rtppacketizer.cpp:122]: You should check the size of `message` for sanity.

**Review by paullouisageneau — COMMENTED** [review]
Thank you, this is a great addition!

Don't worry about the media refactoring, I'll merge this first and include it in the refactoring.

File: src/av1rtppacketizer.cpp
Comment (Sean-Der) [reply src/av1rtppacketizer.cpp]: Fixed! Removed all use of `data` + indexing

File: src/av1rtppacketizer.cpp
Comment (Sean-Der) [reply src/av1rtppacketizer.cpp]: done! Added bound checks everywhere

File: src/av1rtppacketizer.cpp
Comment (Sean-Der) [reply src/av1rtppacketizer.cpp]: Switched! One line is always better then 2 :)

**Comment by Sean-Der** [comment]
All done! Can I get another review please @paullouisageneau 

thanks

File: include/rtc/av1rtppacketizer.hpp
Comment (paullouisageneau) [inline include/rtc/av1rtppacketizer.hpp:31]: Could you please rename `OBU` to `Obu` for consistency with `rtcObuPacketization`, `NalUnit`, `RtpPacketizer`, etc?

File: include/rtc/av1rtppacketizer.hpp
Comment (paullouisageneau) [inline include/rtc/av1rtppacketizer.hpp:49]: Same here

File: include/rtc/description.hpp
Comment (paullouisageneau) [inline include/rtc/description.hpp:53]: It seems this white line was added by mistake.

**Review by paullouisageneau — COMMENTED** [review]
I'd prefer `OBU` to be spelled `Obu` or `obu` in lowercase for consistency, otherwise it looks good to me.

File: include/rtc/av1rtppacketizer.hpp
Comment (Sean-Der) [reply include/rtc/av1rtppacketizer.hpp]: FIxed!

File: include/rtc/av1rtppacketizer.hpp
Comment (Sean-Der) [reply include/rtc/av1rtppacketizer.hpp]: Fixed!

File: include/rtc/description.hpp
Comment (Sean-Der) [reply include/rtc/description.hpp]: Fixed!

**Comment by Sean-Der** [comment]
Thanks again for the review @paullouisageneau :) 

After merging could I have a tag please? This is going right into OBS, it is pretty cool seeing it in action

**Review by paullouisageneau — APPROVED** [review]

**Comment by paullouisageneau** [comment]
> After merging could I have a tag please? This is going right into OBS, it is pretty cool seeing it in action

Great, it is tagged as `v0.19.0-alpha.6`

**Pull request merged**
